src/test/spec/runner/mod.rs

```diff
@@ -2,8 +2,220 @@ mod operation;
 mod test_event;
 mod test_file;
 
+use std::time::Duration;
+
+use crate::{
+    bson::doc,
+    concern::{Acknowledgment, WriteConcern},
+    operation::RunCommand,
+    options::{ClientOptions, CollectionOptions},
+    test::{assert_matches, util::EventClient, TestClient, CLIENT_OPTIONS},
+};
+
 pub use self::{
     operation::AnyTestOperation,
     test_event::TestEvent,
-    test_file::{OperationObject, TestData, TestFile},
+    test_file::{OperationObject, TestCase, TestData, TestFile},
 };
+
+const SKIPPED_OPERATIONS: &[&str] = &[
+    "bulkWrite",
+    "count",
+    "download",
+    "download_by_name",
+    "listCollectionObjects",
+    "listDatabaseObjects",
+    "listIndexNames",
+    "listIndexes",
+    "mapReduce",
+    "watch",
+];
+
+pub async fn run_v2_test(test_file: TestFile) {
+    let has_skipped_op = test_file.tests.iter().any(|test_case| {
```

Comment:
this can be done on a test case level, right? I dont think we want to skip an entire test file because one of the tests had an invalid operation.

Reply:
oh good catch -- this was actually skipping over an operation that needed to be implemented